tests/containers/containerd_rmt.pm

```diff
@@ -25,30 +23,37 @@ sub run {
 
     select_serial_terminal;
     systemctl 'stop firewalld';
+    ensure_ca_certificates_suse_installed();
     install_k3s();
     install_kubectl();
     install_helm();
 
-    my $rmt_helm = get_var('RMTTEST_REPO', 'https://github.com/SUSE/helm-charts/archive/refs/heads/main.tar.gz');
+    my $helm_chart = get_var('HELM_CHART', 'https://github.com/SUSE/helm-charts/archive/refs/heads/main.tar.gz');
     # pull in the testsuite
-    assert_script_run("curl -sSLk $rmt_helm | tar -zxf -");
-    my $rmt_config = get_var('RMTTEST_CONFIG', 'https://gitlab.suse.de/QA-APAC-I/testing/-/raw/master/data/rmtcontainer/myvalue.yaml');
-    assert_script_run("curl -sSLOk $rmt_config");
-    assert_script_run("helm install rmt ./helm-charts-main/rmt-helm -f myvalue.yaml");
+    assert_script_run("curl -sSL $helm_chart | tar -zxf -");
+    my $helm_values = get_var('HELM_CONFIG', 'https://gitlab.suse.de/QA-APAC-I/testing/-/raw/master/data/rmtcontainer/myvalue.yaml');
+    assert_script_run("curl -sSLO $helm_values");
+    my ($repository, $tag) = split(':', get_required_var('CONTAINER_IMAGE_TO_TEST'), 2);
+    my $helm_key = get_required_var('HELM_KEY');
+    my $helm_options = "--debug";
+    assert_script_run("helm install --set ${helm_key}.image.repository=$repository rmt ./helm-charts-main/rmt-helm -f myvalue.yaml $helm_options");
     assert_script_run("helm list");
+    sleep 20;
```

Comment:
Why is this needed? Can we prevent this sleep?

I'm not strictly against using `sleep` if no good other way is possible, but if so we should at least add a comment why this is needed.

Reply:
> Why is this needed? Can we prevent this sleep?
> 
> I'm not strictly against using `sleep` if no good other way is possible, but if so we should at least add a comment why this is needed.

Because it takes some time for the images to download and then run.  I will put a comment.